bin/pgenv

```diff
@@ -20,18 +20,18 @@ PG_DATA=$PGSQL/data
 PGENV_DOWNLOAD_ROOT='https://ftp.postgresql.org/pub/source/'
 
 # Now -w because 9.0 and earlier always time out even when successful.
-PG_CTL="$PGSQL/bin/pg_ctl "
-INITDB="$PGSQL/bin/initdb "
+PG_CTL="$PGSQL/bin/pg_ctl"
+INITDB="$PGSQL/bin/initdb"
 
 #############################################
 # Configuration variable and default settings
 
 PGENV_LOG=$PG_DATA/server.log
 PGENV_PG_USER="postgres"
-PGENV_FLAGS_INITDB="-U $PGENV_PG_USER --locale en_US.UTF-8 --encoding UNICODE"
-PGENV_FLAGS_MAKE="-j3"
-PGENV_FLAGS_CONFIGURE=""
-PGENV_FLAGS_PGCTL_START="-w"
+PGENV_OPTS_INITDB="-U $PGENV_PG_USER --locale en_US.UTF-8 --encoding UNICODE"
```

Comment:
In English, the adjective comes before the subject, so `PGENV_INITDB_OPTS` reads more naturally than `PG_OPTS_INITDB`.